LICENSE

```diff
@@ -1,12 +1,14 @@
-Copyright (c) 2014, The University of North Carolina at Chapel Hill
-All rights reserved.
+Copyright (c) 2023, The iRODS Consortium
```

Comment:
we don't exist as 'The iRODS Consortium'... just legally 'The University of North Carolina at Chapel Hill'